src/assets/scss/components/button.scss

```diff
@@ -1,7 +1,8 @@
 .btn {
+  align-items: center;
   border: 0;
   border-radius: 10px;
-  padding: 8px 16px;
+  padding: 0.5rem;
```

Comment:
This is not fixing the issue on the language button, unfortunately. Plus it is changing our UI. 

You need to consider that this class is use globally, so altering it will alter all buttons. the problem in the language button is that it is inserting an SVG icon rather than a <i> element with the name of the icon. This would fix the problem.